packages/cohttp/cohttp.5.0.0/opam

```diff
@@ -44,7 +44,7 @@ depends: [
   "fmt" {with-test}
   "jsonm" {build}
   "alcotest" {with-test}
-  "crowbar" {with-test}
+  "crowbar" {with-test & >= "0.2"}
```

Comment:
Can you paste here the error that required these extra lower bounds?

Reply:
Added due to:
```
File "cohttp/fuzz/fuzz_header.ml", line 83, characters 28-41:
83 |     let uppercased_letter = Crowbar.range ~min:65 26 in
                                 ^^^^^^^^^^^^^
Error: This function has type int -> int Crowbar.gen
       It is applied to too many arguments; maybe you forgot a `;'.
```